src/annotator/notebook.tsx

```diff
@@ -1,22 +1,24 @@
 import { render } from 'preact';
 
+import type { Destroyable } from '../types/annotator';
 import NotebookModal from './components/NotebookModal';
+import type { NotebookConfig } from './components/NotebookModal';
+import type { EventBus } from './util/emitter';
 import { createShadowRoot } from './util/shadow-root';
 
-/**
- * @typedef {import('../types/annotator').Destroyable} Destroyable
- * @typedef {import('./components/NotebookModal').NotebookConfig} NotebookConfig
- */
+export class Notebook implements Destroyable {
+  private _outerContainer: HTMLElement;
+  public shadowRoot: ShadowRoot;
```

Comment:
`shadowRoot` can be made private as it isn't referenced anywhere else. The tests do access the shadow root, but via `_outerContainer.shadowRoot`.

Reply:
Done